pkg/v1dbschema/schema_test.go

```diff
@@ -129,50 +132,70 @@ func (t *testSchema) TestSchemaV106ToV20x(c *C) {
 	}()
 
 	// create metadata schema.
-	_, err = dbConn.ExecuteSQL(tctx, nil, cfg.Name, []string{
-		`CREATE DATABASE IF NOT EXISTS ` + cfg.MetaSchema,
-	})
+	createSchemaSQL := `CREATE DATABASE IF NOT EXISTS ` + cfg.MetaSchema
```

Comment:
can be removed because it just tested itself

and many rest of them

Reply:
I think so. the original logic is to do prepare jobs so the `UpdateSchema` can work.